slo_generator/backends/cloud_service_monitoring.py

```diff
@@ -357,7 +357,7 @@ def create_slo(self, window: int, slo_config: dict) -> dict:
 
     @staticmethod
     def build_slo(window: int,
-                  slo_config: dict) -> dict:  # pylint: disable=R0912,R0915
+                  slo_config: dict) -> dict:
```

Comment:
Same here. Just move the annotation to line 359 so the linter takes it into account.